components/resources/library/src/commonMain/kotlin/org/jetbrains/compose/resources/FontResources.kt

```diff
@@ -32,4 +33,20 @@ expect fun Font(
     resource: FontResource,
     weight: FontWeight = FontWeight.Normal,
     style: FontStyle = FontStyle.Normal
-): Font
\ No newline at end of file
+): Font
+
+/**
+ * Retrieves the byte array of the font resource.
+ *
+ * @param environment The optional resource environment.
+ * @param resource The font resource.
+ * @return The byte array representing the font resource.
+ */
+@ExperimentalResourceApi
+suspend fun getFontResourceBytes(
+    environment: ResourceEnvironment = getResourceEnvironment(),
```

Comment:
Having `environment` on the first place forces users to use `resource` name explicitly. Is that was the intention?

I don't have a strong opinion here. On one hand, it is better for users to make a conscience choice to use the system environment, on the other hand it is verbose.

Reply:
Yes, it was done to avoid parameters after a vararg in
```kotlin
suspend fun getString(
    environment: ResourceEnvironment,
    resource: StringResource,
    vararg formatArgs: Any
): String
```

Reply:
In this case, I would suggest to do the same for other functions - split them into 2:
```
@ExperimentalResourceApi
suspend fun getFontResourceBytes(
    environment: ResourceEnvironment,
    resource: FontResource
): ByteArray

@ExperimentalResourceApi
suspend fun getFontResourceBytes(
    environment: ResourceEnvironment,
): ByteArray = getFontResourceBytes(getSystemResourceEnvironment(), resource)
```

Otherwise we can't write `getDrawableResourceBytes(Res.drawable.title)`, only `getDrawableResourceBytes(resource = Res.drawable.title)`